docs/changelog/92192.yaml

```diff
@@ -0,0 +1,6 @@
+pr: 92192
+summary: Support for GCS proxies everywhere in the GCS API
```

Comment:
nit: reading this, I find GCS proxy might be a bit misleading. Maybe just support proxies?